draft-ietf-httpbis-header-structure.md

```diff
@@ -305,6 +305,10 @@ For example:
 Example-IntegerHeader: 42
 ~~~
 
+Parsers MUST accurately support integers in the range -9,007,199,254,740,991 to
+9,007,199,254,740,991 inclusive (i.e., -2^53+1 to 2^53-1) at a minimum, and MUST fail parsing on
+values outside this range if they are not supported by the parser.
```

Comment:
I think that this is clearer:

```suggestion
Parsers MUST fail parsing if values are not within the range supported by the parser.
```

I think that's simpler.

That is, say that parsers need to support up to 2^53-1.  Allow more.  Require parsing failure for numbers outside that range (whatever it is).

Reply:
Took a slightly different approach.